Extension/ASPNETCore-EssentialJS2-Extension/Visual-Studio/Syncfusion-Project-Templates.md

```diff
@@ -76,7 +76,7 @@ The following steps is used to create the **Syncfusion ASP.NET Core (Essential J
 
 4. Click Create, the Syncfusion ASP.NET Core (Essential JS 2) Application has been created.
 
-    ![css refernce](images/readme-file.PNG)
+    ![css refernce for Extension ASP.NET Core essentialjs2 with syncfusion project templates](images/aspnet-core-essentialjs2-extension-with-project-templates-created-application.PNG)

```

Comment:
Don't use "Extension ASP.NET Core essentialjs2 with syncfusion project template" the word in alt text and image name. Instead of use "ASP.NET Core project template" ina ll places.

Reply:
Resolved